lib/fog/azurerm/models/compute/virtual_machine_sizes.rb

```diff
@@ -0,0 +1,18 @@
+module Fog
+  module Compute
+    class AzureRM
+      class VirtualMachineSizes < Fog::Collection
+        attribute :location
+        model Fog::Compute::AzureRM::VirtualMachineSize
+
+        def all(async = false)
+          requires :location
+          virtual_machine_sizes = service.list_virtual_machine_sizes(location, async).map do |virtual_machine_size|
```

Comment:
```suggestion
          virtual_machine_sizes = service.list_virtual_machine_sizes(location, async).map do |virtual_machine_size|
```

Update this line of code accordingly as well.